src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -192,6 +202,9 @@ public void configMotors() {
 		launcherAngleTwoMotor.getEncoder().setPosition(launcherAngleEncoder.getPosition());
 		launcherAngleTwoMotor.getEncoder().setPositionConversionFactor(PIVOT_GEARING_RATIO);
```

Comment:
@kirbt  - these lines (and the ones above) duplicate what the new `zeroRelativeEncoder()` method does. Since the relative encoder appears to support resetting its position, I'd recommend using that instead of keeping track of an offset & manually using that. And we really should look at the robot to see what the PIVOT_GEARING_RATIO value ought to be. It appears to be assuming a 3x3x5 gearbox and a 16 tooth sprocket driving a 64 tooth gear.

Reply:
So, `zeroRelativeEncoder()` could just call `launcherAngleOneMotor.getEncoder().setPosition(...)` with the passed-in position, assuming that there hasn't been meaningful movement between when the reference value was calculated & when the method is called.

Reply:
Resolving, as this isn't critical to fix since it just means the motor's relative encoder has a somewhat random position offset - which is generally the case since it always depends on where the arm is when power is first applied.